lib/demo.feature

```diff
@@ -36,6 +36,8 @@ Feature: demo features
     When maximize window
     When get window size
     When resize window to 123 width 123 height
+    When switch to iframe by id: "123"
```

Comment:
@TopcuKloia  switch to iframe by  "123" id yapsak

Reply:
@sinemkorkmaz-kloia yapayım, sanırım önceden yazılmışlarla daha uyumlu olacak gibi. Ama gramer olarak biraz aksak geldi nedense.